tests/page-objects/default/contacts/contacts.wdio.page.js

```diff
@@ -83,6 +84,44 @@ const deathCardSelectors = {
   deathPlace: () => $(`${DEATH_CARD_TEST_ID} div[test-id="contact.profile.death.place"] p.card-field-value`),
 };
 
+const INMUNIZATION_CARD_TEST_ID = `div[test-id="contact.profile.immunizations"]`;
+const inmunizationCardSelectors = {
+  inmunizationCard: () => $(INMUNIZATION_CARD_TEST_ID),
+};
```

Comment:
It's only used in one place; we can remove it for now. 
```suggestion
const inmunizationCardSelectors = {
  inmunizationCard: () => $('div[test-id="contact.profile.immunizations"]'),
};
```